ui/components/issuessection/issuessection.go

```diff
@@ -222,10 +233,14 @@ func (m *Model) FetchSectionRows() tea.Cmd {
 	if m == nil {
 		return nil
 	}
-	m.Issues = nil
-	m.Table.ResetCurrItem()
-	m.Table.Rows = nil
-	m.IsLoading = true
+
+	if !m.Cursor.HasNextPage {
+
+		m.Issues = nil
+		m.Table.ResetCurrItem()
+		m.Table.Rows = nil
+		m.IsLoading = true
+	}
 	var cmds []tea.Cmd
 	cmds = append(cmds, m.CreateNextTickCmd(spinner.Tick))
```

Comment:
I don't think we need this when we fetch the next rows, as this spinner is showing only on initial render, when we don't have any rows yet

Reply:
When ```m.IsLoading``` is set to ```true``` while fetching the new rows there is a flash behavior (list disappears and appears again), so I excluded setting it while fetching the next rows